src/victorops.coffee

```diff
@@ -231,8 +235,8 @@ class VictorOps extends Adapter
     # remove keys from the victor ops incident keys list and from the brain
     # if they are older than 24 hours
     voIKeysFiltered = voIKeys.filter((item) ->
-      if new Date(item.timestamp).addDays(1) < new Date
-        @robot.brain.remove item.name
+      if new Date(item.timestamp).getDate() + 1 < new Date
+        #@robot.brain.remove item.name
```

Comment:
Seems like it should be fine. I'll test and update once I confirm that there are no hidden gotchas.